### ISSUE TITLE
Unresponsive GUI

### ISSUE THREAD

**Issue by SandroVA** [bluelightzero/unresponsive-cursor#7]
Having the cursor change and display the reason for disconnecting isn't ideal for me. I would much prefer some sort of GUI. Is there any chance that you could add this feature?

**Comment by bluelightzero** [comment]
Do you mean an overlay? (A little box you can place in corners and stuff.)

![image](https://user-images.githubusercontent.com/4492982/110317996-d82d3d00-8004-11eb-8836-1dbd9e8ef9a9.png)

**Comment by SandroVA** [comment]
Yes sorry. I tried looking for another plugin that gave an overlay like this but couldn't find any.

**Comment by jmdev91** [comment]
> Yes sorry. I tried looking for another plugin that gave an overlay like this but couldn't find any.

Sounds like you might want Ping Graph. It gives an overlay that shows spikes whenever lag is occurring, along with some current numbers for context.
